crates/core/tedge_api/src/workflow/supervisor.rs

```diff
@@ -150,6 +132,16 @@ impl WorkflowSupervisor {
         }
     }
 
+    /// Mark the current version of an operation workflow as being in use.
+    ///
+    /// Return the current version if any.
+    pub fn use_current_version(&mut self, operation: &OperationName) -> Option<WorkflowVersion> {
+        match self.workflows.get_mut(&operation.as_str().into()) {
+            Some(versions) => versions.use_current_version().cloned(),
+            None => None,
+        }
```

Comment:
```suggestion
       self.workflows.get_mut(&operation.as_str().into())?.use_current_version().cloned()
```

Reply:
Good point. I will fixed that in a follow-up PR.